### ISSUE TITLE
Exhaust smoke from all US tanks and tank destroyers is emitted in wrong location/direction

### ISSUE THREAD

**Issue by handsm** [DarklightGames/DarkestHour#689]
For some reason the exhaust smoke is emitted sideways on the right, just drifting out of the hull. I'm pretty sure these all exhausted from the rear, so I believe this is just a simple bug in the set up that needs fixing.

**Comment by handsm** [comment]
Firefly exhaust fixed as part of commit https://github.com/DarklightGames/DarkestHour/commit/3bfa8a42cbf2

**Comment by handsm** [comment]
Fixed in commit https://github.com/DarklightGames/DarkestHour/commit/1f1458bc7a754c84d54d08f31d2835e14296dc38.

None of the exhaust/tail pipes on these vehicles is actually modelled, i.e. shown in the vehicle mesh. I have positioned the exhaust effects on the location they should be, with a suitable exhaust direction depending on whether it's emitting via an exhaust deflector. Have set up one or two emitters (three in one case) according to number of exhausts, but some vehicles have twin exhausts next to each other and in those cases I've only used one emitter as there's no point having two on the same location.

Sherman M4A1 - Had two exhausts just under the upper rear hull plate, centrally and next to each other, so have used one emitter. Exhausts back and downwards, without a deflector.

Sherman M4A3 - Had two exhausts either side of the engine door, but are covered by a large grille deflector. Set up two emitters blowing out of the left and right grille sections, directed backwards and slightly down.

Sherman Firefly (M4A4) - Had two exhausts under the upper rear hull plate, one on each side. Set up two emitters blowing out of the upper rear hull 'hood', directed downwards and slightly away, more or less in line with angle of the upper rear hull plate.

Wolverine/Achilles (built on M4A2 Sherman chassis) - Had a pair of exhausts centrally , where the engine doors would be on other Shermans (model in game shows engine doors but it is inaccurate). Hidden by a simple exhaust deflector on our model. Used one emitter, positioned centrally and blowing out of the deflector backwards and down a bit. Have previously changed the exhaust emitter to be a diesel smoke effect, as the M4A2 was the Sherman with the twin diesel engines.

Jackson (built on M4A3 Sherman chassis) - Had two exhausts either side of the engine doors, but in our model they are covered by an unusual deflector, which looks like it 'boxes in' the exhaust smoke more than deflecting it. Assume the smoke drifts out around the edges of the box, which would still serve the purpose of stopping it blasting into the ground and sending up clouds of dust. Set up two emitters, either side of the box, and tried to make it look like exhaust smoke is drifting out it around the edges.

Hellcat - As far as I can tell, exhaust smoke was emitted upwards along the grilles running along the back end of the engine deck. Seen videos of running Hellcats that show that happening. There is an exhaust manifold below that grille and I believe it has multiple outlets vented upwards. Have set up three emitters, one for each grille, with smoke directed upwards. The ideal might be for an alternative exhaust emitter for upwards directed exhausts like this, with the smoke drifting upwards more slowly, at least when the engine is idling.

Stuart - Had two exhausts covered by a grille on the bottom of the rear hull (grille not modelled in game), spread out and offset to the right. Used two emitters, directed back and slightly downwards.

Greyhound - Had an exhaust tucked under the rear right mudguard plate, where it meets the rear hull. Later models had the tail pipe higher up and poking through a hole in the mudguard, but believe our model is the earlier version. Exhaust was more or less correctly positioned on this vehicle, but have tweaked a little and directed the smoke backwards and slightly down.
